app/models/course.rb

```diff
@@ -20,6 +20,7 @@
 #  created_at              :datetime         not null
 #  updated_at              :datetime         not null
 #  changed_at              :datetime         not null
+#  recruitment_cycle_id    :integer
```

Comment:
nullable?

Reply:
Ya, this will have to be nullable for the first migration at least, unless we give it a default value (in the SQL), which is probably not the best. Otherwise:

1. Add `course.recruitment_cycle_id`, nullable
2. Populate `recruitment_cycle_id` for all the courses with a data migration
3. Make `course.recruitment_cycle_id` not nullable.

Reply:
aint that the default?

`Add2019RecruitmentCycleToCourses`

Reply:
We can't make it not null until we've ran the data migration so I suggest doing a follow up PR for this.

Reply:
Guess I got baited to reject this, will fall for it everytime.